profiles/coreos/targets/generic/make.defaults

```diff
@@ -98,3 +98,6 @@ ACCT_USER_SYSTEMD_RESOLVE_ID=245
 # tss seems to be one of those users with a mismatching UID/GID
 ACCT_GROUP_TSS_ID=252
 ACCT_USER_TSS_ID=236
+
+# Disable creation of /usr/lib as symlink
+SYMLINK_LIB="no"
```

Comment:
Could we switch to 17.1 profile instead?

Reply:
Maybe, I'm not sure what other changes this implies. My idea was to limit the changes for now and only align the amd64 board with the arm64 board and leave the SDK out.

Reply:
i would love it if we could make the jump to 17.1 independently, but I acknowledge that it's more work.